js/directives/lucca/user-picker.js

```diff
@@ -29,7 +29,7 @@
 	}]; // MAGIC LIST OF PROPERTIES
 
 	var uiSelectChoicesTemplate = "<ui-select-choices position=\"down\" repeat=\"user in users\" refresh=\"find($select.search)\" refresh-delay=\"0\" ui-disable-choice=\"!!user.overflow\">" +
-	"<div ng-bind-html=\"user.firstName + ' ' + user.lastName | highlight: $select.search\" ng-if=\"!user.overflow\"></div>" +
+	"<div>{{user.firstName}} {{user.lastName}} <span class=\"lui label secondary\" ng-if=\"user.info\">{{user.info}}</span></div>" +
```

Comment:
add the `luifHighlight` filter in the `user-picker.js` file because it depends on ui-select via the highlight filter. do not add it to the `genericFilters.js` file even if it is its place, otherwise the dependencies will be all over the place